### PR TITLE
Fix bugs to enable module to be usable as an npm module

### PR DESCRIPTION
**Pull request by andyzg**
To test:
- Install the module using `npm install @codeium/react-code-editor` in a React app or in the `demo/` folder of this folder
- Import CodeiumEditor from the module
- Default layout is 100% width and 300px height
- No branding changes or API restrictions were made
- Preview:

image

Remaining:
- There's a bug that happens upon completion. Needs more investigation
image

### CODE REVIEW

File: demo/app/page.tsx [hunk 1]
```diff
@@ -6,7 +6,9 @@ import { CodeiumEditor } from "@codeium/react-code-editor";
 export default function Home() {
   return (
     <main className={styles.main}>
-      <CodeiumEditor language="python" />
+      <div>
+        <CodeiumEditor language="python" />
+      </div>
```
Comment (khou22) [inline demo/app/page.tsx]: Does it have to nested in a `div` now?

File: src/components/CodeiumEditor/CodeiumEditor.tsx [hunk 2]
```diff
@@ -77,12 +77,20 @@ export const CodeiumEditor: React.FC<CodeiumEditorProps> = (props) => {
     defaultValue: getDefaultValue(props.language),
   };
 
+  const layout = {
```
Comment (khou22) [inline src/components/CodeiumEditor/CodeiumEditor.tsx:85]: Worth adding these usage caveats to the function docstring

**Comment by andyzg** [comment]
@khou22 packed two fixes into the recent changes along with addressing comments:
- Catch network error for AcceptComplete
- Fix logo not showing if the Editor is not at the top

**Review by khou22 — APPROVED** [review]
Nice!

**Review by khou22 — APPROVED** [review]
Skimmed through, looks reasonable

**Pull request merged**